teamcity/resource_build_trigger_schedule.go

```diff
@@ -260,6 +314,51 @@ func expandTriggerScheduleOptions(d *schema.ResourceData) (*api.TriggerScheduleO
 	return opt, nil
 }
 
+func expandCronSchedule(v []interface{}) (*api.CronSchedule, error) {
+	raw := v[0].(map[string]interface{})
+	var cronSchedule api.CronSchedule
+
+	if v, ok := raw["seconds"]; ok {
+		cronSchedule.Seconds = v.(string)
+	}
+	if v, ok := raw["minutes"]; ok {
+		cronSchedule.Minutes = v.(string)
+	}
+	if v, ok := raw["hours"]; ok {
+		cronSchedule.Hours = v.(string)
+	}
+	if v, ok := raw["day_of_month"]; ok {
+		cronSchedule.DayOfMonth = v.(string)
+	}
+	if v, ok := raw["month"]; ok {
+		cronSchedule.Month = v.(string)
+	}
+	if v, ok := raw["day_of_week"]; ok {
+		cronSchedule.DayOfWeek = v.(string)
+	}
+	if v, ok := raw["year"]; ok {
+		cronSchedule.Year = v.(string)
+	}
+	// Validate the cron expression
+	cronParser := cron.NewParser(cron.Second | cron.Minute | cron.Hour | cron.Dom | cron.Month | cron.Dow)
```

Comment:
Just to confirm, this package has identical (or at least is a proper subset) of the configuration supported by TeamCity?

Reply:
From my investigation it does, at least for the parsing bit we care about here. Documentation [here](https://github.com/robfig/cron)

Reply:
SGTM, thanks for confirming!